gcp/modules/gpii-flowmanager/dns.tf

```diff
@@ -0,0 +1,20 @@
+data "terraform_remote_state" "network" {
+  backend = "gcs"
+
+  config {
+    credentials    = "${var.serviceaccount_key}"
+    bucket         = "${var.project_id}-tfstate"
+    prefix         = "${var.env}/infra/network"
+    encryption_key = "/dev/null"
```

Comment:
What happened to the original comment?

Does not look like there is any progress on that TF issue:
https://github.com/hashicorp/terraform/issues/18933

Reply:
Hm, I don't have that good understanding of it, but I was thinking it's not the case - because `/infra/network` is encrypted with Google-managed key, we actually want this to be null to avoid using the env variable we have set - `TF_VAR_key_tfstate_encryption_key`, independently whether that works or not, and that the described issue affects only scenarios with Customer-supplied keys. Happy to add the comment if that's not the case?